src/main/java/org/jenkinsci/plugins/dockerbuildstep/cmd/remote/CreateImageRemoteCallable.java

```diff
@@ -36,8 +38,9 @@
     Map<String, String> buildArgsMap;
     boolean noCache;
     boolean rm;
-
-    public CreateImageRemoteCallable(Config cfgData, Descriptor<?> descriptor, String expandedDockerFolder, String expandedImageTag, String dockerFileRes, Map<String, String> buildArgsMap, boolean noCache, boolean rm) {
+    BuildListener listener = null;
```

Comment:
explicit initialization is not needed (`= null`)

Reply:
Agreed.  

 I removed the initialization
 I assigned the passed in listener.